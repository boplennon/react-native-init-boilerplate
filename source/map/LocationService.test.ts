import LocationService from './LocationService';

describe('components', () => {
  describe('LocationService', () => {
    it('should match snapshot', () => {
      expect(LocationService).toMatchSnapshot();
    });
  });
});
