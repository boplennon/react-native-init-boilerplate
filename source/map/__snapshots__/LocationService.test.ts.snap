// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components LocationService should match snapshot 1`] = `
LocationServiceImplementation {
  "getCurrentLocation": [Function],
  "getMapPointForCoordiantes": [Function],
}
`;
